modules/dxp/apps/segments/segments-asah-connector-test/src/testIntegration/java/com/liferay/segments/asah/connector/internal/messaging/test/InterestTermsCheckerTest.java

```diff
@@ -131,7 +131,7 @@ public void testCheckIndividualSegments() throws Exception {
 						).toString())));
 
 			ReflectionTestUtil.invoke(
-				_interestTermsChecker, "checkInterestTerms",
+				_interestTermsMessageListener, "_checkInterestTerms",
```

Comment:
```_interestTermsMessageListener.receive(Message message)```

call it instead of reflection using a `Message` with the required `userId` property